gramps/gen/filters/_genericfilter.py

```diff
@@ -53,7 +58,8 @@
 class GenericFilter:
     """Filter class that consists of several rules."""
 
-    logical_functions = ["or", "and", "xor", "one"]
+    logical_functions = ["and", "or", "one"]
+    table = "person"
```

Comment:
I could not find where `table` is used?
Also defined in derived classes

Reply:
I think you are correct. I rewrote some generic code, and so this is no longer needed.